assets/js/reducers/questionnaire.validation.js

```diff
@@ -204,6 +204,12 @@ const validateSmsLangPrompt = (
     addError(context, `${path}.sms`, k("SMS prompt must not be blank"), lang, "sms")
     return
   }
+  const smsParts = prompt.sms && prompt.sms.trim().split(smsSplitSeparator) || []
+  smsParts.forEach((part, idx) => {
+    if(isBlank(part)){
+      addError(context, `${path}.sms`, k(`[${idx}]SMS prompt must not be blank`), lang, "sms")
```

Comment:
The `k` function is there to tag strings that are translatable. In this case, given that the `[${idx}]` part is a prefix for the code itself, I wouldn't show it to translators (they could choose to move the preffix to be a suffix, or whatever.

If it was a parameter, we should parameterize it like we do below:
https://github.com/instedd/surveda/blob/c046534ac3857275576fee9d031239557f0aadc3/assets/js/reducers/questionnaire.validation.js#L306

But in this case, I think we should leave the index part out of the `k` key:

```javascript
      addError(context, `${path}.sms`, `[${idx}]${k("SMS prompt must not be blank")}`, lang, "sms")
```

It'd be even better if we didn't abuse the notation with a prefix (and, instead, make it an object or an array). But that'd be even more complicated.